src/components/ProjectView/ImageCarousel.vue

```diff
@@ -114,7 +96,7 @@ const getImageSrc = (src) => {
     class="thumbnail__carousel__container"
   >
     <div
-      v-for="(item, index) in data"
+      v-for="(item, index) in store.state.smallImageCache"
```

Comment:
oops I didn't think about that and agreed. I thought it would be cleaner to not use a prop if we already had the data in store but it makes more sense for the future to use the store in the parent component and pass it down to the children.